src/errors.ts

```diff
@@ -59,6 +45,20 @@ export function ensureErrorType(err: unknown): Error & Partial<AzFuncError> {
     }
 }
 
+export function setErrorMessage(err: Error, message: string): Error & Partial<AzFuncError> {
+    try {
+        err.message = message;
+        return err;
+    } catch (e) {
+        if (e instanceof TypeError) {
+            // Return original error if we can't set the message
+            return err;
+        } else {
+            throw e;
+        }
+    }
```

Comment:
I think we should ignore all errors, not just TypeError. Also, you're modifying the original `err` directly, so it would be simpler to just return void

```suggestion
    try {
        err.message = message;
    } catch (e) {
        // ignore
    }
```

Reply:
Agreed. I have made the suggested changes.